app/views/claims/support/schools/mentors/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<% render "claims/support/primary_navigation", current: :organisations %>
+<% content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_support_school_mentors_path(@school)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= t(".header") %> <%= @school.name %></span>
+      <h2 class="govuk-heading-l"><%= @mentor.full_name %></h2>
+
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key"><%= t(".attributes.mentors.first_name") %></dt>
+          <dd class="govuk-summary-list__value"><%= @mentor.first_name %></dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key"><%= t(".attributes.mentors.last_name") %></dt>
+          <dd class="govuk-summary-list__value"><%= @mentor.last_name %></dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key"><%= t(".attributes.mentors.trn") %></dt>
+          <dd class="govuk-summary-list__value"><%= @mentor.trn %></dd>
```

Comment:
I think it is better to use `govuk_summary_list` component. It will take care of the html for you and is a bit easier to read. 

Also, I _think_ we've decided (based on other PRs) to use eg `Mentor.human_attribute_name(:first_name)` rather than add these all to a translation file. The exception would be trn because that will be formatted in different ways depending on the context.